### PR TITLE
Extract `greeting` into `OutputDocument`

### PR DESCRIPTION
**Pull request by stevenwilkin**
Addresses #79

### CODE REVIEW

File: app/domain/output_document.rb [hunk 1]
```diff
@@ -58,6 +63,8 @@ def variant
   end
 
   def greeting
```
Comment (gjvis) [inline app/domain/output_document.rb]: `greeting` doesn't seem appropriate for this text. Perhaps `lead` would be better? (or `lede`)

Comment (stevenwilkin) [inline app/domain/output_document.rb]: +1 for `lead`

File: spec/domain/output_document_spec.rb [hunk 2]
```diff
@@ -129,8 +129,10 @@ def excludes_all_circumstances
     end
   end
 
-  describe '#greeting' do
-    it 'needs details'
+  describe '#lead' do
+    subject { output_document.lead }
+
+    it { is_expected.to eq('You recently had a Pension Wise guidance appointment with A Guider from The Pensions Advisory Service on March 30, 2015') }
```
Comment (andrewgarner) [inline spec/domain/output_document_spec.rb]: "A Guider" isn't representative test data - it should be a first name.

The date should be formatted as "30 March 2015" as per the GOV.UK style guide.

Comment (stevenwilkin) [inline spec/domain/output_document_spec.rb]: The date format will be addressed with #81

I'll update the guider name

File: app/templates/output_document_ineligible.html.erb [hunk 3]
```diff
@@ -11,12 +11,7 @@
   <p>Dear <%= attendee_name %>
     ,</p>
 
-  <p class="emphasize">You recently had a Pension Wise guidance appointment
-    with <%= guider_name %> from
-
-    <%= guider_organisation %>
-
-    on <%= appointment_date %>.</p>
+  <p class="emphasize"><%= lead %>.</p>
```
Comment (andrewgarner) [inline app/templates/output_document_ineligible.html.erb]: This line doesn't have a punctuation mark in the artwork. We should remove it.

Comment (stevenwilkin) [inline app/templates/output_document_ineligible.html.erb]: Sorted

**Pull request closed without merging**